src/mainboard/clevo/mtl-h/variants/igpu/overridetree.cb

```diff
@@ -1,4 +1,16 @@
 chip soc/intel/meteorlake
+	# Power limits,
+	# PsysPL2, PsysPL3, PL4 are configured by EC at runtime
+	register "power_limits_config[MTL_P_682_482_CORE]" = "{
+		.tdp_pl1_override = 35,
+		.tdp_pl2_override = 64,
+		.tdp_pl4 = 120,
```

Comment:
Not sure if pl4 can be bigger than pmax. I know EC will limit it, but I don't know what the consequences may be.

How much Watts does the power supply have?

Reply:
PSU is 90W, but Pmax has to be 99W because that's the scale that the battery charger uses to report Psys to CPU.

I thought PL4 can be up to PSU rating + max battery rating but it feels like every doc I read about it says something different. Will adjust PL4 to the same value as PMAX

Reply:
https://github.com/Dasharo/coreboot/commit/ed3fdaec053cf201ed16a54a564b7455416b9bdf#diff-5440c50c876ba495f3ff1099c44d020892a1941250b8987c8aecea3d3e47d80fR7

Reply:
Fair enough